src/ngx_http_lua_util.c

```diff
@@ -4327,5 +4364,36 @@ ngx_http_lua_escape_log(u_char *dst, u_char *src, size_t size)
     return 0;
 }
 
+ngx_int_t ngx_http_lua_copy_escape_header(ngx_http_request_t *r,
```

Comment:
Bad coding style. Forgot to run the `ngx-releng` tool?

Reply:
ngx-releng won't complain when return type and function name at the same line